packages/attestor/light-client/src/header.rs

```diff
@@ -10,39 +9,27 @@ pub struct Header {
     pub new_height: u64,
     /// Timestamp of the new height
     pub timestamp: u64,
-    /// Opaque serde-encoded data that was signed
+    /// ABI-encoded attestation data that was signed (bytes32[] for packet commitments or `StateAttestation` struct)
     pub attestation_data: Vec<u8>,
-    /// Signatures of the attestors
-    pub signatures: Vec<Signature>,
-    /// Public keys of the attestors submitting attestations
-    pub pubkeys: Vec<VerifyingKey>,
+    /// Raw 65-byte signatures in (r||s||v) format for ECDSA address recovery
+    pub signatures: Vec<Vec<u8>>,
```

Comment:
Technically, we might be able to use `alloy_primitives::Signature` here, not sure if it's worth it?